lambda_function.py

```diff
@@ -1,9 +1,161 @@
+
+        response = self.cloudformation_client.create_stack_instances(
+            StackSetName=self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME,
+            Accounts=[self.customer_account_id],
+            Regions=[self.region_name],
+            ParameterOverrides=[{'ParameterKey': 'AccountRoleName', 'ParameterValue': self.customer_account_new_role_name},
+                                {'ParameterKey': 'Externalid', 'ParameterValue': self.customer_account_external_id}],
+            OperationPreferences={
+                'FailureToleranceCount': 0,
+                'MaxConcurrentCount': 3,
+            })
+
+        self.wait_for_stack_operation(response["OperationId"])
+
+    def wait_for_stack_operation(self, operation_id: str) -> None:
+        for retry_count in range(self.STACK_OPERATION_WAIT_RETRIES):
+            response = self.cloudformation_client.describe_stack_set_operation(
+                StackSetName=self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME, OperationId=operation_id)
+
+            if response["StackSetOperation"].get("Status") == "SUCCEEDED":
+                return response["StackSetOperation"]["StackSetId"]
+
+            logger.info(f"Current operation status {response['StackSetOperation'].get('Status')}, going to sleep for {self.STACK_OPERATION_WAIT_SLEEP}")
+            time.sleep(self.STACK_OPERATION_WAIT_SLEEP)
+
+    def delete_stack_instances(self) -> Dict:
+        logger.info(f"Deleting stack instance for StackSet: {self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}, "
+                    f"region: {self.region_name}, AccountId: {self.customer_account_id}")
+
+        response = self.cloudformation_client.delete_stack_instances(StackSetName=self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME,
+                                                                Regions=[self.region_name], RetainStacks=True,
+                                                                Accounts=[self.customer_account_id])
+        return response
+
+    def delete_stack_set(self):
+        logger.info(f"Deleting StackSet: {self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}")
+        return self.cloudformation_client.delete_stack_set(StackSetName=self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME)
+
+    def register_to_dome9(self) -> Dict:
+        dome9_client = Client()
+        credentials = CloudAccountCredentials(arn=self.customer_account_new_role_arn, secret=self.customer_account_external_id)
+        payload = CloudAccount(name=self.customer_account_name, credentials=credentials)
+        resp = dome9_client.aws_cloud_account.create(body=payload)
+        return resp
+
+    def create_stack_set_flow(self) -> None:
+        try:
+            self.create_stack_set()
+        except Exception as e:
+            if "NameAlreadyExistsException" in repr(e):
+                logger.info(f"Received NameAlreadyExistsException for stack_set '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}'. Error: {repr(e)}")
```

Comment:
isn't it `exception` level?

Reply:
I though to do it. But it's ignored now - both exceptions got to regular stdout without formatting.